osgi/jta/src/main/resources/jbossts-properties.xml

```diff
@@ -90,7 +98,6 @@
     -->
     <entry key="RecoveryEnvironmentBean.recoveryModuleClassNames">
         com.arjuna.ats.internal.arjuna.recovery.AtomicActionRecoveryModule
-        com.arjuna.ats.internal.txoj.recovery.TORecoveryModule
```

Comment:
It's transactional objects for Java recovery module. TX4j is a narayana extension so possibly not a default usage for you